server/build/webpack.js

```diff
@@ -154,11 +154,12 @@ export default async function createCompiler (dir, { dev = false, quiet = false
         if (!(/\.js$/.test(interpolatedName))) {
           return { content, sourceMap }
         }
-
+        const babelRuntimePath = require.resolve('babel-runtime/package')
+          .replace(/[\\/]package\.json$/, '')
         const transpiled = babelCore.transform(content, {
           presets: [require.resolve('babel-preset-es2015')],
           sourceMaps: dev ? 'both' : false,
-          // Here we need to resolve styled-jsx/style to the absolute paths.
+          // Here we need to resolve all modules to the absolute paths.
           // Earlier we did it with the babel-preset.
           // But since we don't transpile ES2015 in the preset this is not resolving.
           // That's why we need to do it here.
```

Comment:
I think that's because it's added by babel. In my tests I had no issues with other modules.